tests/Feature/Platform/Services/PlayerGameActivityServiceTest.php

```diff
@@ -32,12 +32,12 @@ public function testActivity(): void
         $game = $this->seedGame(achievements: 5, withHash: false);
 
         // ==== Start session at time1 ====
-        $now = Carbon::now()->floorSecond();
-        $time1 = $now->clone()->subMinutes(100);
+        $now = Carbon::now()->floorSeconds();
+        $time1 = $now->clone()->subMinutes(100)->startOfSecond();
```

Comment:
Recommend using `startOfSecond` when calculating now so you don't have to apply it to each derived value. Besides, isn't that what `floorSeconds` is supposed to be doing?

Reply:
Fixed in latest.